src/Components/Activities/ActivitiesCards/ActivitiesCards.js

```diff
@@ -0,0 +1,11 @@
+import ActivityCard from '../ActivityCard/ActivityCard';
+const ActivitiesCards = ({activities}) => {
+
+    return (
+        <div>
+            {activities ? activities.map(e=><ActivityCard props={e} key={e.id}/>) : <p>No hay Actividades</p>}
```

Comment:
This is not a blocker, but for future components texts like "No hay Actividades" can be on a constants file so they can be reused or replaced easily.

Reply:
instead of creating a new component it can be a function on Activities Pagination